simpleviewinc/keg-hub
# Build fixes

## Context
* autocomplete is not working b/c it didn't have the right jsutils version
* keg-components rollup build has unresolved dependency issues due to the way `reStyle` is imported
* the package will not start due to the NODE_ENV being set to `local`

## Goal
* Ensure keg-components can build and run with retheme reStyle and the latest jsutils and keg-cli updates

## Updates
* `repos/keg-components/package.json`
  * added `NODE_ENV` to the storybook commands so that the package could be run
  * updated `jsutils` version to `8.5.0`
* `repos/keg-components/src/components/form/autocomplete/autocomplete.js` && `repos/keg-components/src/components/form/scrollable/item/item.js` && `repos/keg-components/src/components/form/scrollable/select/selectView.js` && `repos/keg-components/src/theme/reStyle.examples.js` && `repos/keg-components/src/theme/reStyle.stories.mdx`
  * updated import to fix unresolved dependencies


## Testing
* `keg dpg run package=ghcr.io/simpleviewinc/keg-components:build-fixes`
* verify that the autocomplete function works (specifically, the dropdown list of items)
* attach to the image
* `yarn build`
* verify it builds with no errors or warnings
